src/cluster/replication.cc

```diff
@@ -75,6 +75,11 @@ Status FeedSlaveThread::Start() {
 
   if (s) {
     t_ = std::move(*s);
+
+    // Re-enable the bufferevent and set up callbacks after detachment
+    auto bev = conn_->GetBufferEvent();
+    bufferevent_enable(bev, EV_READ);
+    bufferevent_setcb(bev, &FeedSlaveThread::staticReadCallback, nullptr, nullptr, this);
```

Comment:
In `this->loop`, we send data to file descriptor directly in a different thread. Here we can using `bufferevent` to read from the file descriptor in the callback. I think it is ok, because in the callback we only read, and in the loop, we only write. But I am not very familiar with this lib and C..